### PR TITLE
Allow use of existing secret for pulsar manager credentials

### PR DESCRIPTION
**Pull request by pinkavaj**
### Motivation

This allows store secrets out of the configuration file and manage them with external tools to keem them actualy secret.

### Modifications

Create the pulsar manager secret only conditionaly.

### Verifying this change

Tested on my own deployment.

- [ ] Make sure that the change passes the CI checks.

### CODE REVIEW

**Comment by jiazhai** [comment]
@pinkavaj Seems there are some ci failure, would you please help take a look?

**Comment by pinkavaj** [comment]
> @pinkavaj Seems there are some ci failure, would you please help take a look?

I would like to, but it looks like this is some CI issue `Error: Kubernetes cluster unreachable`

**Comment by sijie** [comment]
@jiazhai the CI is failing due to apache/pulsar-helm-chart#76

**Comment by sijie** [comment]
@pinkavaj Can you rebase your pull request to the latest master?

File: charts/pulsar/values.yaml [hunk 1]
```diff
@@ -963,6 +963,8 @@ pulsar_manager:
     hostname: ""
     path: "/"
 
+  ## If set use existing secret with specified name to set pulsar admin credentials.
+  existingSecret:
```
Comment (sijie) [inline charts/pulsar/values.yaml:967]: Can you name it as `existingSecretName`?

Comment (pinkavaj) [reply charts/pulsar/values.yaml]: Updated. It was my opinio the original name is the most used for this purpose and thus best fit into the helm charts ecosystem, but this of course might be only mi bias, sinc I does not have any strong support for this..

**Review by wolfstudy — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**